recipe-server/normandy/base/tests/test_api.py

```diff
@@ -0,0 +1,82 @@
+import json
+
+import pytest
+
+from django.conf.urls import url
+from django.views.generic import View
+
+from normandy.base.api.views import APIRootView
+
+
+@pytest.mark.django_db
+class TestApiRoot(object):
```

Comment:
This test is more about actually having the expected api. Since part of our public api is (now) having certain names here, I think it is important to have this test. I'll also add more unit-like tests for the router.